tasks.yaml

```diff
@@ -35,6 +36,15 @@ tasks:
     actions:
       - task: setup:k3d-test-cluster
 
+  - name: registry-login
+    actions:
+      - task: common-setup:registry-login
+        with:
+          registry: ${REGISTRY}
+          registryUsername: ${REGISTRY_USERNAME}
+          registryPassword: ${REGISTRY_PASSWORD}
+          registryRetryInterval: ${REGISTRY_RETRY_INTERVAL}
```

Comment:
I think I recall this being problematic if you don't pass in a `REGISTRY_RETRY_INTERVAL` var? It ends up nulling out the default, so `sleep` errors out with no seconds set.

Reply:
```suggestion
```

Reply:
good catch!
rather than remove, i am thinking we may want to tweak this number in the future if/when we encounter failures.  so i am adding back the `--set` in the CI job.